### PR TITLE
Initial Commit

### PR DESCRIPTION
**Pull request by maximede**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# wcc-to-acs-replicator
+
+
+develop: [![Build Status](https://travis-ci.org/uw-it-edm/wcc-to-acs-replicator.svg?branch=develop)](https://travis-ci.org/uw-it-edm/wcc-to-acs-replicator)
+
```
[the remark this answers is not in the document]
Comment (maximede) [reply README.md]: I'm thinking of using codacy for code coverage, would that work for you ?

File: src/main/java/edu/uw/edm/wcctoacsreplicator/clients/contentapi/ContentApisClient.java [hunk 2]
```diff
@@ -161,17 +167,17 @@ private String getContentApi2CreateOrUpdateURL(Document document, Optional<WCCTo
...
     }
 
-    private InputStreamResource getContentApiFileStream(String contentId, Document document) throws IOException {
+    private InputStreamResource getContentApiFileStream(String contentId, Document document, String lastModifier) throws IOException {
         URL url = new URL(replicatorProperties.getContentApi().getHostAndContext() + CONTENT_API_GET_V_3_FILE + contentId + CONTENT_API_GET_PRIMARY_RENDITION_PARAMS);
 
         HttpURLConnection urlConnection = (HttpURLConnection) url.openConnection();
 
         urlConnection.setRequestMethod(HttpMethod.GET.name());
         //TODO which user ?
```
Comment (agagne) [inline src/main/java/edu/uw/edm/wcctoacsreplicator/clients/contentapi/ContentApisClient.java]: remove TODO

File: src/main/java/edu/uw/edm/wcctoacsreplicator/clients/contentapi/ContentApisClient.java [hunk 3]
```diff
@@ -128,6 +131,10 @@ public void createNewOrNewRevision(String contentId, Document document) throws R
 
         String acsId = response.getBody().getId();
 
+
+        /**
+         * This should be removed once ACS starts sending the create events to SNS
```
Comment (agagne) [inline src/main/java/edu/uw/edm/wcctoacsreplicator/clients/contentapi/ContentApisClient.java]: add as 'TODO:'

File: src/main/java/edu/uw/edm/wcctoacsreplicator/clients/contentapi/ContentApisClient.java [hunk 4]
```diff
@@ -142,7 +149,7 @@ public void createNewOrNewRevision(String contentId, Document document) throws R
 
 
             log.trace("got response {}", response.getStatusCode());
-            log.trace("" + response.getBody());
+            log.trace("{}" + response.getBody());
```
Comment (agagne) [inline src/main/java/edu/uw/edm/wcctoacsreplicator/clients/contentapi/ContentApisClient.java]: should have a comma: `log.trace("{}" , response.getBody());`

File: src/main/java/edu/uw/edm/wcctoacsreplicator/clients/contentapi/ContentApisClient.java [hunk 5]
```diff
@@ -0,0 +1,209 @@
...
+    }
+
+    public void update(String contentId, Document document) throws RestClientException, IOException, NoMappingForIdException {
+
+        log.trace("received create for {}", contentId);
+
+        Optional<WCCToACSMapping> mappingForWCCId = wccToACSMappingService.getMappingForWCCId(contentId);
+
+        if (!mappingForWCCId.isPresent()) {
+            //TODO what to do ? Call create ?
```
Comment (agagne) [inline src/main/java/edu/uw/edm/wcctoacsreplicator/clients/contentapi/ContentApisClient.java]: remove TODO

**Pull request merged**